internal/upf/ebpf/xdp/utils/gtp.h

```diff
@@ -30,6 +30,11 @@
 #include "xdp/utils/parsers.h"
 #include "xdp/utils/trace.h"
 
+volatile const int n3_vlan;
+volatile const int n3_vlan = 0;
```

Comment:
Is there a reason we set VLAN's to 0 as the zero-value here but to NULL when we write to `ctx`?

Reply:
Here it is an integer, and it represents the VLAN ID. The value 0 is reserved by the spec as meaning that there is no VLAN, and in C it evaluates to false. This serves as the configuration for the VLAN ID that should be inserted in outgoing packets.

`ctx->vlan` on the other hand is a pointer to a `vlan_hdr` struct, and it points to the start of the vlan header in the received packet if we want to read it. Setting it to NULL is the same thing as setting to 0, but it is a bit clearer as it is a pointer.